lib/ansible/cli/galaxy.py

```diff
@@ -1203,7 +1203,7 @@ def _execute_install_collection(
         if len([p for p in collections_path if p.startswith(path)]) == 0:
             display.warning("The specified collections path '%s' is not part of the configured Ansible "
                             "collections paths '%s'. The installed collection won't be picked up in an Ansible "
-                            "run." % (to_text(path), to_text(":".join(collections_path))))
+                            "run, unless within a playbook adjacent collections directory." % (to_text(path), to_text(":".join(collections_path))))
```

Comment:
This wording doesn't really clarify the issue. It's probably easier to change the "won't" to "may".

Reply:
How does this sound? 

`The specified collections path '%s' is not part of the configured Ansible collections paths '%s'. The installed collection may not be picked up in an Ansible run, unless within a playbook adjacent collections directory.`

Reply:
The phrase "unless within a playbook adjacent collections directory" is a mouthful. It's also not immediately apparent what that means unless the reader is well versed in Ansible jargon.

> The specified collections path '%s' is not part of the configured Ansible collections paths '%s'. The installed collection may not be picked up in an Ansible run.

If there is a strong desire to mention adjacency (I know this was explicitly requested in the related issue, but I don't think it's the clearest phrasing), then something like this is probably best:

> The specified collections path '%s' is not part of the configured Ansible collections paths '%s'. The installed collection will not be picked up in an Ansible run unless adjacent to a playbook.